fili-security/src/main/java/com/yahoo/bard/webservice/web/security/RoleDimensionApiFilterRequestMapper.java

```diff
@@ -41,7 +41,7 @@
  * This mapper is intended to route based on user-category style roles such as routing super-user access to a
  * different mapping chain.
  */
-public class RoleDimensionApiFilterRequestMapper extends ChainingRequestMapper<DataApiRequest> {
+public class RoleDimensionApiFilterRequestMapper extends ChainingRequestMapper<DataApiRequestImpl> {
```

Comment:
Having the implementation class here seems incorrect. I think it should be the interface.